de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorHandler.java

```diff
@@ -116,10 +113,10 @@ public Editor openEditor(
         @NotNull
             IProject project, boolean activate) {
 
-        IResource resource = virtualFileConverter
-            .getResource(virtualFile, project);
+        IResource resource = VirtualFileConverter
+            .convertToResource(virtualFile, project);
 
-        if (resource == null) {
+        if (resource == null || !SessionUtils.isShared(resource)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYM7E0EcSutaxLwSa&open=AWcxYM7E0EcSutaxLwSa)

[ISSUE_KEY]: AWcxYM7E0EcSutaxLwSa